common/src/main/public/development.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "environment": "DEVELOPMENT",
-  "odbURI": "wss://lucuma-odb-development.herokuapp.com/ws",
+  "odbURI": "wss://lucuma-odb-no-sharing.herokuapp.com/ws",
```

Comment:
Do we need to change this to merge?

Reply:
Yes, but we also need to merge the no-sharing branch of the ODB.